app/views/layouts/travel_insurance_admin.html.erb

```diff
@@ -0,0 +1,49 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="MAS Retirement Adviser Directory Administration">
+  <meta name="author" content="Team A">
+  <title>Travel Insurance Administration</title>
+  <%= stylesheet_link_tag 'admin' %>
+  <%= csrf_meta_tags %>
+</head>
+
+<body>
+
+<nav class="navbar navbar-inverse navbar-fixed-top">
+  <div class="container">
+    <div class="navbar-header">
+      <a class="navbar-brand" href="#">Travel Insurance Administration</a>
+    </div>
+    <div id="navbar" class="collapse navbar-collapse">
+      <ul class="nav navbar-nav">
+        <%= active_link_to 'Travel Insurance Firms', admin_travel_insurance_firms_path, wrap_tag: 'li' %>
+      </ul>
+    </div>
+  </div>
+</nav>
+
+<div class="container">
+  <% if flash.notice %>
+    <div class="t-flash-message alert alert-warning" role="alert">
+      <%= flash.notice %>
+    </div>
+  <% end %>
+
+  <%= yield %>
+</div>
+
+<footer class="footer">
+  <div class="container">
+    <p class="text-muted">Money Advice Service <%= DateTime.now.year %></p>
+  </div>
+</footer>
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
+<%= javascript_include_tag('admin') %>
+</body>
+</html>
```

Comment:
I wonder if it's worth DRYing up the commonality between the two admin layouts? Fine to leave it if you think it's too early to bother.

Reply:
It is easy to move so I created a partial and move the admin main content there.